### PR TITLE
{tools}[GCCcore/11.3.0] Perl v5.34.1, expat v2.4.8, ncurses v6.3, ...

### PR DESCRIPTION
**Pull request by bartoldeman**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by bartoldeman** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
bear-pg0105u36b.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/89d565d9a2ee005f69fb41cf39f14028 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/f7190e145039a69a9d21169ef578fcde for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
bear-pg0306u07a.bear.cluster - Linux RHEL 8.5, POWER, 8335-GTX (power9le), 4 x NVIDIA Tesla V100-SXM2-16GB, 470.57.02, Python 3.6.8
See https://gist.github.com/efa9c63710b24aef09f905c8247ff52a for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2
CORE_CNT=16

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
skylake-alma8-eb - Linux AlmaLinux 8.5, x86_64, Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz (skylake), Python 3.6.8
See https://gist.github.com/aae11b192a07996c7d2110c82cea58d0 for a full test report.

File: easybuild/easyconfigs/l/libreadline/libreadline-8.1-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'libreadline'
+version = '8.1'
```
[the remark this answers is not in the document]
Comment (bartoldeman) [reply easybuild/easyconfigs/l/libreadline/libreadline-8.1-GCCcore-11.3.0.eb:4]: ah correct. Will do!
I checked https://tiswww.case.edu/php/chet/readline/rltop.html but it's out of date.

**Comment by bartoldeman** [comment]
@boegelbot please test @ generoso

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2
CORE_CNT=16

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
skylake-alma8-eb - Linux AlmaLinux 8.5, x86_64, Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz (skylake), Python 3.6.8
See https://gist.github.com/670cc430fca302f61411d5f1f2556b9d for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
zen2-ubuntu-eb - Linux Ubuntu 22.04, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.10.4
See https://gist.github.com/3a069405cc617a5ad49b482d3376da00 for a full test report.

**Review by SebastianAchilles — APPROVED** [review]
Checked all Perl extensions with a script. All are up to date :-)

lgtm

**Comment by SebastianAchilles** [comment]
Going in, thanks @bartoldeman!

**Pull request merged**